static/js/components/Dashboard.js

```diff
@@ -2,9 +2,14 @@
 import React from 'react';
 import CourseList from './CourseList';
 import UserImage from './UserImage';
+import _ from 'lodash';
 
 class Dashboard extends React.Component {
 
+  formatId(id) {
+    return id ? `MMM${_.padStart(id.toString(), 6, '0')}` : undefined;
```

Comment:
ok, never mind the previous comment: if you "pretty print" the number in the backend, there is not reason to repeat the job in the front-end.
I think this should be done in the backend and here you just put the string